packages/webpack-config-terra/lib/postcss/ThemePlugin.js

```diff
@@ -30,22 +30,26 @@ module.exports = postcss.plugin('terra-theme-plugin', (config) => {
     return acc;
   }, []);
 
-  return (root) => {
-    if (defaultThemeSelector || themesToRemove.length) {
-      root.walkRules((node) => {
-        // Scrub css modules pseudo classes from the selector
-        const selector = removeCssModulesPseudoClasses(node.selector);
-        // Clone the default theme node and apply as root.
-        if (selector === defaultThemeSelector) {
-          const clone = node.clone({ selector: ':root' });
-          root.append(clone);
-        }
+  return {
+    postcssPlugin: 'terra-theme-plugin',
+    Once (root) {
+      if (defaultThemeSelector || themesToRemove.length) {
+        root.walkRules((node) => {
+          // Scrub css modules pseudo classes from the selector
+          const selector = removeCssModulesPseudoClasses(node.selector);
+          // Clone the default theme node and apply as root.
+          if (selector === defaultThemeSelector) {
+            const clone = node.clone({ selector: ':root' });
+            root.append(clone);
+          }
 
-        // Remove the undesired theme node from it's parent.
-        if (themesToRemove.includes(selector)) {
-          node.parent.removeChild(node);
-        }
-      });
+          // Remove the undesired theme node from it's parent.
+          if (themesToRemove.includes(selector)) {
+            node.parent.removeChild(node);
+          }
+        });
+      }
     }
-  };
-});
+  }
+};
+module.exports.postcss = true
```

Comment:
not a big deal, but i had to dig a little to see what this meant
```suggestion
module.exports.postcss = true // distinguish between require('plugin') and require('plugin')(opts) end-user calls
```

Reply:
I was just following the plugin upgrade guide 😅